themes/jaeger-docs/layouts/partials/docs/header.html

```diff
@@ -47,16 +47,16 @@
       {{ end }}
       {{ if $isLatest }}
       <span class="tag is-medium is-success">Latest</span>
-      {{ else }}
+      {{ else if .GetPage $latestUrl}}
```

Comment:
so this checks if the page actually exists in the website?

Reply:
Yes

Reply:
🚀